tests/unit/test_fulfillManifest_process.py

```diff
@@ -32,27 +32,26 @@ def __init__(self):
 
     def test_runProcess(self, testProcess, mocker):
         runMocks = mocker.patch.multiple(
-            FulfillProcess,
-            getManifests=mocker.DEFAULT,
+            FulfillURLManifestProcess,
+            fetch_and_update_manifests=mocker.DEFAULT,
         )
 
         testProcess.runProcess()
 
-        runMocks['getManifests'].assert_called_once()
+        runMocks['fetch_and_update_manifests'].assert_called_once()
 
 
-    def test_getManifests_success(self, testProcess, mocker):
-        processMocks = mocker.patch.multiple(FulfillProcess,
+    def test_fetch_and_update_manifests(self, testProcess, mocker):
+        processMocks = mocker.patch.multiple(FulfillURLManifestProcess,
             load_batches=mocker.DEFAULT,
             getObjectFromBucket=mocker.DEFAULT,
-            update_manifest=mocker.DEFAULT
+            update_metadata_object=mocker.DEFAULT
         )
 
         mockPrefix = mocker.MagicMock(prefix='testPrefix')
         mockTimeStamp = mocker.MagicMock(timeStamp='testTimeStamp')
         
-        testProcess.getManifests(mockTimeStamp)
-
+        testProcess.fetch_and_update_manifests(mockTimeStamp)
 
         processMocks['load_batches'].assert_called_once_with('testPrefix','test_aws_bucket')
         
```

Comment:
nit: new line!

Reply:
There is a new line after the `processMocks` method call.